.gitmodules

```diff
@@ -16,3 +16,6 @@
 [submodule "kwiz-dotfiles"]
 	path = kwiz-dotfiles
 	url = https://github.com/kwizapp/kwiz-dotfiles
+[submodule "wiki"]
```

Comment:
Add the wiki as a submodule to make editing easier (pull/push instead of web ide).

@alexscheitlin @nikzaugg make sure to clone the new submodule by running the update command.